sites/public/src/forms/filters/FilterForm.tsx

```diff
@@ -1,357 +1,268 @@
-import {
-  AvailabilityFilterEnum,
-  EnumListingFilterParamsStatus,
-} from "@bloom-housing/backend-core/types"
+import { EnumListingFilterParamsStatus } from "@bloom-housing/backend-core/types"
 import {
   t,
-  SelectOption,
   Form,
-  Select,
   Field,
   Button,
   AppearanceStyleType,
   FrontendListingFilterStateKeys,
   ListingFilterState,
+  GridSection,
+  GridCell,
+  AppearanceBorderType,
 } from "@bloom-housing/ui-components"
 import { useForm } from "react-hook-form"
 import { Region } from "@bloom-housing/ui-components/src/helpers/regionNeighborhoodMap"
-
-const isValidZipCodeOrEmpty = (value: string) => {
-  // Empty strings or whitespace are valid and will reset the filter.
-  if (!value.trim()) {
-    return true
-  }
-  let returnValue = true
-  value.split(",").forEach((element) => {
-    if (!/^[0-9]{5}$/.test(element.trim())) {
-      returnValue = false
-    }
-  })
-  return returnValue
-}
+import { useEffect, useState } from "react"
+import axios from "axios"
+import { listingFeatures } from "@bloom-housing/shared-helpers"
 
 interface FilterFormProps {
   onSubmit: (data: ListingFilterState) => void
+  onClose: (isOpen: boolean) => void
   filterState?: ListingFilterState
 }
 
+interface optionInterface {
+  value: string
+  label: string
+  translation?: string
+}
+
+const getTranslationString = (str: string) => {
```

Comment:
we could shorten this to something like `str.indexOf(“Bdrm”) ? str.replace (“Bdrm”, “Plus”) : str.push(“Plus)`